pyproject.toml

```diff
@@ -43,29 +58,101 @@ source = [
    ".tox/pypy*/site-packages",
 ]
 
-[tool.black]
-skip-magic-trailing-comma = true
+[tool.pytest.ini_options]
+minversion = "8.2"
+asyncio_mode = "auto"
 
-[tool.isort]
-profile = "black"
-known_first_party = ["aiofiles"]
+[tool.ruff]
+exclude = [
```

Comment:
We already pass the dirs to ruff in the Makefile, do we need this bunch of excludes here?

Reply:
You are right.